src/modules/LanItems.rb

```diff
@@ -1320,7 +1320,7 @@ def bridge_slaves(master)
 
     # Creates item's startmode human description
     #
-    # @param item_id [Integer] a key for {#Items}
+    # @param ifcfg [Hash] Interface sysconfig configuration
```

Comment:
consistency note. Parameter carrying device's sysconfig is usually named ```devmap```. ```ifcfg``` is usually used for configuration name (```ifcfg == eth0``` --> file with sysconfig is ```ifcfg-eth0```)